packages/forklift-console-plugin/src/modules/Providers/views/details/components/CredentialsSection/components/edit/OvirtCredentialsEdit.tsx

```diff
@@ -71,17 +71,21 @@ export const OvirtCredentialsEdit: React.FC<EditComponentProps> = ({ secret, onC
 
   const insecureSkipVerifyHelperTextPopover = (
     <Trans t={t} ns="plugin__forklift-console-plugin">
-      {
-        "Note: If 'Skip certificate validation' is selected, migrations from this provider will not be secure.<br><br> Insecure migration means that the transferred data is sent over an insecure connection and potentially sensitive data could be exposed."
-      }
+      Note: If <strong>Skip certificate validation</strong> is selected, migrations from this
+      provider will not be secure.
+      <br />
+      <br /> Insecure migration means that the transferred data is sent over an insecure connection
+      and potentially sensitive data could be exposed.
     </Trans>
   );
 
   const cacertHelperTextPopover = (
     <Trans t={t} ns="plugin__forklift-console-plugin">
-      {
-        'Note: Use the Manager CA certificate unless it was replaced by a third-party certificate, in which case use the Manager Apache CA certificate. <br><br>You can retrieve the Manager CA certificate at:<br>'
-      }
+      Note: Use the Manager CA certificate unless it was replaced by a third-party certificate, in
+      which case use the Manager Apache CA certificate. <br />
+      <br />
+      You can retrieve the Manager CA certificate at:
+      <br />
```

Comment:
```suggestion
      You can retrieve the Manager CA certificate at:
```

Reply:
much nicer to display external link in a new line.
I did change this line to:
`You can retrieve the Manager CA certificate at:<br />`

Reply:
ok